src/elements/content-uploader/OverallUploadsProgressBar.js

```diff
@@ -76,15 +77,13 @@ const OverallUploadsProgressBar = ({
     isVisible,
     isExpanded,
     hasMultipleFailedUploads,
+    customPrompt,
```

Comment:
Since you're making changes in here, do you mind alphabetizing the props?

Reply:
done